Userland/Libraries/LibGfx/ImageFormats/ExifReader.cpp

```diff
@@ -195,6 +195,46 @@ ErrorOr<void> ExifReader::read_tag()
         VERIFY(!TRY(read_idf(exif_value.get<u32>())).has_value());
     }
 
+    if (tag == 271) {
```

Comment:
Might be nice to put these magic numbers in an enum/enum class. Though they're only used here and the context makes them clear enough, so it's no big deal.